pandas/core/nanops.py

```diff
@@ -1240,7 +1305,7 @@ def nancorr(a, b, method="pearson", min_periods=None):
     return f(a, b)
 
 
-def get_corr_func(method):
+def get_corr_func(method: str):
     if method in ["kendall", "spearman"]:
         from scipy.stats import kendalltau, spearmanr
     elif callable(method):
```

Comment:
can you add another conditions which checks that method in ['kendall', 'spearman', 'kendall'] *only*

Reply:
> can you add another conditions which checks that method in ['kendall', 'spearman', 'kendall'] _only_

Just to be sure, did you mean:
```python
["kendall", "spearman", "pearson"]
```
?

Reply:
yes